### PR TITLE
Networking V2: add ability to update FIP port's fixed IP

### LINKED ISSUE #1425: Networking V2: add possibility to update FIP port's fixed IP
Floating IP update method should support changing the port's fixed IP .

Neutron API code: https://github.com/openstack/neutron/blob/mitaka-eol/neutron/extensions/l3.py#L146

This filed could be changed to another IP, or omitted. Empty or null should not be supported.

Ref https://github.com/terraform-providers/terraform-provider-openstack/issues/646

### PR DESCRIPTION
**Pull request by kayrus**
Resolves #1425

### CODE REVIEW

**Comment by kiwik** [comment]
recheck

**Comment by kayrus** [comment]
recheck

**Comment by kayrus** [comment]
lbaas CRUD goes crazy. @jtopjian ready for review.

**Comment by kayrus** [comment]
@jtopjian kindly ping

File: openstack/networking/v2/extensions/layer3/floatingips/requests.go [hunk 1]
```diff
@@ -124,6 +124,7 @@ type UpdateOptsBuilder interface {
 type UpdateOpts struct {
 	Description *string `json:"description,omitempty"`
 	PortID      *string `json:"port_id,omitempty"`
+	FixedIP     string  `json:"fixed_ip_address,omitempty"`
```
Comment (jtopjian) [inline openstack/networking/v2/extensions/layer3/floatingips/requests.go]: `FixedIPAddress` - the field should always match the JSON tag.

Comment (kayrus) [reply openstack/networking/v2/extensions/layer3/floatingips/requests.go]: I took it from [this](https://github.com/gophercloud/gophercloud/blob/df38e1611dbe76a187bbcec07797714af9671066/openstack/networking/v2/extensions/layer3/floatingips/requests.go#L18) place

Comment (jtopjian) [reply openstack/networking/v2/extensions/layer3/floatingips/requests.go]: Ah - good catch. Looks like this package a few other tag inconsistencies.

Let's keep it at `FixedIP` then. I'll merge this in the morning. Thanks!

**Pull request merged**